technologies_deployments.md

```diff
@@ -49,15 +49,15 @@ limitations under the License.
             <div style="min-height:140px; padding: 10px;">
               Python is supported with Matplotlib, Conda, Pandas SQL and PySpark integrations.
             </div>
-            <a href="/docs/0.7.2/interpreter/python.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
+            <a href="/docs/latest/interpreter/python.html" class="panel-button">USE NOW <span class="glyphicon glyphicon-chevron-right"></span></a>
           </label>
         </div>
       </div>
     </div>
     <div class="col-md-12 col-sm-12 col-xs-12 text-center">
       <p class="bottom-text">
         See more details in Zeppelin supports 20+ different interpreters.
-        <a href="/docs/0.8.0-SNAPSHOT/manual/interpreters.html">LEARN MORE <span class="glyphicon glyphicon-chevron-right" style="font-size:15px;"></span></a>
+        <a href="/supported_interpreters.html">LEARN MORE <span class="glyphicon glyphicon-chevron-right" style="font-size:15px;"></span></a>
```

Comment:
doesn't need "/docs/latest/" here?

Reply:
supported_interpreters.html is in root directory of `gh-pages` branch, instead of docs directory of master. (part of website, not part of zeppelin documentation). So the path is correct.